build-tools/update

```diff
@@ -41,6 +39,7 @@ configBuf = configBuf.replace(/\/\* version \*\/[^;\n]*;/, `/* version */ Config
 
 fs.writeFileSync('config/config.js', configBuf);
 console.log("DONE");
+eval(configBuf);
```

Comment:
It would probably be better to do

```js
const context = {Config: {}}
require('vm').runInNewContext(configBuf, context);
const {Config} = context;
```

Reply:
Nice! Thanks!